1649817310635	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1649817310918	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile 304
1649819043402	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1649819043402	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1649819043406	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1649819043453	Sync.SyncScheduler	DEBUG	Woke from sleep.
1649819043453	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1649819043453	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1649819043454	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1649819045454	Sync.Service	DEBUG	User-Agent: LibreWolf/99.0 (Linux x86_64) FxSync/1.101.0.20220410100435.desktop
1649819045454	Sync.Service	INFO	Starting sync at 2022-04-13 03:04:05 in browser session MMupkoldrupu
1649819045454	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1649819045457	Sync.Status	INFO	Resetting Status.
1649819045457	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1649819045458	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1649819045458	FirefoxAccounts	DEBUG	getOAuthToken enter
1649819045458	FirefoxAccounts	TRACE	getCachedToken returning cached token
1649819045458	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1649819045459	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1649819045459	Sync.SyncAuthManager	INFO	Getting sync key
1649819045459	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1649819045459	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1649819045459	FirefoxAccounts	DEBUG	getOAuthToken enter
1649819045460	FirefoxAccounts	TRACE	getCachedToken returning cached token
1649819045460	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1649819045460	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1649819045460	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1649819045460	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1649819045460	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1649819045461	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1649819045461	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1649819045461	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1649819045461	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1649819045461	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1649819045462	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1649819049919	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1649819049919	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1649819049920	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1649819049920	Sync.Service	DEBUG	User-Agent: LibreWolf/99.0 (Linux x86_64) FxSync/1.101.0.20220410100435.desktop
1649819049920	Sync.Service	INFO	Starting sync at 2022-04-13 03:04:09 in browser session MMupkoldrupu
1649819049921	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1649819049921	Sync.Service	INFO	Cannot start sync: already syncing?
1649819051187	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1649819051415	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1649819051415	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1649819051416	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1649819051417	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1649819051429	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
